hw/xfree86/compat/devices.c

```diff
@@ -0,0 +1,29 @@
+#include <dix-config.h>
+
+#include <X11/Xfuncproto.h>
+
+#undef Ones
+
+/*
+ * this is specifically for NVidia proprietary driver: they're again lagging
+ * behind a year, doing at least some minimal cleanup of their code base.
+ * All attempts to get in direct contact with them have failed.
```

Comment:
maybe add a comment that it's for the old driver, and the new is not needing them anymore